Stratis.Bitcoin/Features/BlockStore/BlockStoreLoopStep.cs

```diff
@@ -42,32 +69,46 @@ protected BlockStoreLoopStep(BlockStoreLoop blockStoreLoop)
             this.BlockStoreLoop = blockStoreLoop;
         }
 
+        /// <inheritdoc />
```

Comment:
`<inheritdoc />` is used on derived classes, the comment should be present in base/abstract classes or interfaces, so that derived classes can use `<inheritdoc />` to refer to the comment as "inherited". Here you have inherit, but from what?